scripts/test.mjs

```diff
@@ -0,0 +1,28 @@
+import esbuild from "esbuild";
+import fs from "fs";
+import path from "path";
+
+function findTestFiles(dir, files = []) {
+  fs.readdirSync(dir, { withFileTypes: true }).forEach((dirent) => {
+    const res = path.resolve(dir, dirent.name);
+    if (dirent.isDirectory()) {
+      findTestFiles(res, files);
+    } else if (dirent.name.endsWith(".test.ts")) {
```

Comment:
Eh the esbuild name uses `.test.bundle.ts` and if it turned into `bundle.test.ts` this might be confusing to debug.